packages/shared/providers.yaml

```diff
@@ -1386,6 +1386,50 @@ coupa-compass:
             description: The domain of your Coupa Compass account
             format: hostname
 
+databricks-account:
+    display_name: Databricks(Account Level)
+    categories:
+        - analytics
+    auth_mode: OAUTH2_CC
+    token_url: https://accounts.cloud.databricks.com/oidc/accounts/${connectionConfig.accountId}/v1/token
+    token_params:
+        grant_type: client_credentials
+    proxy:
+        base_url: https://accounts.cloud.databricks.com/api/2.0/accounts/${connectionConfig.accountId}
+    docs: https://docs.nango.dev/integrations/all/databricks
+    connection_config:
+        accountId:
+            type: string
+            title: Account ID
+            description: The ID to your account
+            pattern: '^[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{12}$'
+            example: 123e4567-e89b-12d3-a456-426614174000
+
+databricks-workspace:
+    display_name: Databricks(Workspace Level)
+    categories:
+        - analytics
+    auth_mode: OAUTH2_CC
+    token_url: https://${connectionConfig.databricksInstance}/oidc/v1/token
+    token_params:
+        grant_type: client_credentials
+    proxy:
+        base_url: https://${connectionConfig.workspaceUrl}/api/2.0/
+    docs: https://docs.nango.dev/integrations/all/databricks
+    connection_config:
+        databricksInstance:
+            type: string
+            title: Databricks Instance
+            description: The instance to your databricks deployment
+            pattern: '^[a-z0-9_-]+(\.[a-z0-9_-]+)*$'
+            example: dbc-a1b2345c-d6e7.cloud.databricks.com
+        workspaceUrl:
+            type: string
+            title: Workspace URL
+            description: The base workspace URL to your account
+            pattern: '^[a-z0-9_-]+(\.[a-z0-9_-]+)*$'
+            example: dbc-a1b2345c-d6e7.cloud.databricks.com
```

Comment:
Why it's two different params that seems to be similar? Also are they always ending with `.cloud.databricks.com` or it can be hosted?

Reply:
This can be hosted on Amazon AWS, Microsoft Azure, and Google Cloud Platform which can change the shape of the `base_url`. From the docs, they have the following as an example; `https://<databricks-instance>/api/2.0/workspace/get-status`.